.env.demo.compose

```
@@ -565,3 +565,42 @@
+  "responseTypesSupported": ["code"],
+  "scopesSupported": ["mcp.read", "mcp.write", "mcp.admin", "openid", "profile", "email"],
+  "codeChallengeMethodsSupported": ["S256"]
+}]'
+
+# Alternative: Token introspection for opaque tokens
+# MCP_AUTH_INTROSPECTION_ENDPOINT=https://authdemo.gauzy.co/oauth2/introspect
+# MCP_AUTH_INTROSPECTION_CLIENT_ID=gauzy-mcp-demo-server
+# MCP_AUTH_INTROSPECTION_CLIENT_SECRET=demo-client-secret
+
+# Cache settings for performance
+MCP_AUTH_TOKEN_CACHE_TTL="300"     # 5 minutes
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)